kolibri/plugins/coach/assets/src/views/index.vue

```diff
@@ -59,6 +61,12 @@
         if (this.pageName === constants.PageNames.EXAMS) {
           return 'exams-page';
         }
+        if (this.pageName === constants.PageNames.CREATE_EXAM) {
```

Comment:
See https://github.com/learningequality/kolibri/pull/1117/files#r108287946

If these are associated with route parameters, you might be able to test with those, but iirc, one of @indirectlylit 's motivations locking everything into a single route was to reduce complexity of all the `names` -- route name vs page name vs scoped pagenames (assigned in `pageState`), etc. I dunno. worth thinking about.